pkg/state/keys.go

```diff
@@ -390,6 +390,21 @@ func (k *disabledAliasKey) bytes() []byte {
 	return buf
 }
 
+func (k *disabledAliasKey) unmarshal(data []byte) error {
+	if len(data) != disabledAliasKeySize {
+		return errInvalidDataSize
+	}
+	if data[0] != disabledAliasKeyPrefix {
+		return errInvalidPrefix
+	}
+	var err error
+	k.alias, err = proto.StringWithUInt16Len(data[1:])
```

Comment:
```k.alias, err := proto.StringWithUInt16Len(data[1:])```

Reply:
Won't work here, `err` have to be declared beforehand because only assignment can be used to assign value to `k.alias`.